content/ja/handbook/legal/esg/index.md

```
@@ -0,0 +1,371 @@
+upstream_path: "/handbook/legal/esg/"
+upstream_sha: "02cf85a2ba59858c59b2a31a0356f2371a2a8979"
+translated_at: "2026-04-30T00:55:37Z"
+translator: claude
+stale: false
+---
+
+## クイックリンク
+
+* GitLab におけるサステナビリティの詳細については、[LevelUp](https://levelup.edcast.com/insights/ECL-627ad07b-b21d-4044-a109-34b12e711988) および[ハンドブック](/handbook/legal/esg/esg-training/)から ESG トレーニングコースを受講してください
+* [#sustainability-esg](https://gitlab.enterprise.slack.com/archives/CPWSJBCDP) Slack チャンネルに参加して、更新情報の確認、質問、フィードバックの提供を行ってください
+* お客様から ESG 関連の質問がありますか？以下の[お客様 ESG 質問へのサポートリクエスト](#requesting-support-with-customer-esg-questions)セクションの手順に従ってください
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)